src/panels/LogPanel/ScriptLogEntry.tsx

```diff
@@ -39,12 +41,12 @@ export function ScriptLogEntry({ script, index, isSelected, onToggleSelection }:
         </Text>
       </Code>
       <CopyToClipboardButton value={script} />
-      <Tooltip label={'Rerun script'}>
+      <Tooltip label={t('aria-labels.select-script')}>
```

Comment:
```suggestion
      <Tooltip label={t('aria-labels.rerun-script')}>
```
?

Reply:
Have been thinking about that these copy-psate-errors happens sooo easily. I hope we haven't spread too many of those around 😓

Reply:
I've been trying to keep an eye out as well, but its always possible